RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -808,6 +849,10 @@ void MuonIdProducer::fillMuonId(edm::Event& iEvent, const edm::EventSetup& iSetu
    for ( const auto& chamber : info.chambers )
    {
      if (chamber.id.subdetId() == 3 && rpcHitHandle_.isValid()  ) continue; // Skip RPC chambers, they are taken care of below)
+
+     if (doME0_){
```

Comment:
Yes, please remove the ``` if (doME0_) ```. 

By the specific ME0 block, I meant the code that follows after ``` // Fill ME0 info ``` on line 974 and below.